source/UIAHandler/__init__.py

```diff
@@ -904,23 +1141,49 @@ def _isUIAWindowHelper(self,hwnd):
 						and AllowUiaInChromium.getConfig() != AllowUiaInChromium.YES  # Users can prefer to use UIA
 					)
 				):
+					if isDebug:
+						log.debug("_isUIAWindowHelper:Chromium window treated as non-UIA")
 					return False
 			elif windowClass == "ConsoleWindowClass":
-				return utils._shouldUseUIAConsole(hwnd)
+				if not utils._shouldUseUIAConsole(hwnd):
+					if isDebug:
+						log.debug("Windows console treated as non-UIA")
+					return False
+			if isDebug:
+				log.debug("Treating as UIA")
+		else:
+			if isDebug:
+				log.debug("window does not have UIA server side provider. Treating as non-UIA")
 		return bool(res)
 
-	def isUIAWindow(self,hwnd):
+	def isUIAWindow(self, hwnd: int, isDebug=False) -> bool:

```

Comment:
Add extra type information.

```suggestion
	def isUIAWindow(self, hwnd: int, isDebug: bool = False) -> bool:
```